pkg/kubelet/apis/config/types.go

```diff
@@ -75,6 +75,22 @@ const (
 	PodTopologyManagerScope = "pod"
 )
 
+type GracefulNodeShutdownPodPolicyType string
+
+const (
+	// GracefulNodeShutdownPodPolicySetTerminal is Graceful Node Shutdown policy type,
+	// which will ensure that during graceful node shutdown, pods will be terminated by kubelet and
+	// placed into terminal Failed phase on API server,
+	// and as a result, will not start again if nodes becomes ready again.
```

Comment:
nit: clarify its a particular node we're talking about becoming ready again

```suggestion
	// and as a result, will not start again if the node becomes ready again.
```

Reply:
Fixed, thanks.